app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -2,8 +2,10 @@
   meta ||= []
   navigation ||= []
   with_border ||= false
+  classes = %w(gem-c-layout-footer govuk-footer)
+  classes << "gem-c-layout-footer--border" if with_border
 %>
-<footer class="gem-c-layout-footer govuk-footer <%= "gem-c-layout-footer--border" if with_border %> role="contentinfo">
+<footer class="<%= classes.join(' ') %>" role="contentinfo">
```

Comment:
Non-blocker, suggestion: if we use the tag helper we get the `join` for 'free' and the attribute sanitised.
```rb
<%= tag.footer class: css_classes, role: "contentinfo" do %>
   ...
<% end %>

Reply:
THAT was the thing I meant to include as part of this change, thanks @alex-ju. Added now 🤦